Source/WebKit/UIProcess/Extensions/WebExtensionContext.h

```diff
@@ -193,8 +193,10 @@ class WebExtensionContext : public API::ObjectImpl<API::Object::Type::WebExtensi
     void didFailNavigation(WKWebView *, WKNavigation *, NSError *);
     void webViewWebContentProcessDidTerminate(WKWebView *);
 
+#ifdef __OBJC__
     _WKWebExtensionContext *wrapper() const { return (_WKWebExtensionContext *)API::ObjectImpl<API::Object::Type::WebExtensionContext>::wrapper(); }
```

Comment:
Would be nice to use `dynamic_objc_cast<_WKWebExtensionContext>(...)` here from `<wtf/cocoa/TypeCastsCocoa.h>`.